.github/workflows/basic.yml

```diff
@@ -29,6 +29,15 @@ jobs:
         env:
           RUST_BACKTRACE: 1
 
+      - name: Run tests w/o default features
+        uses: actions-rs/cargo@v1
```

Comment:
Just a heads up that we need to stop using `actions-rs/cargo`, I made an issue: https://github.com/DA0-DA0/dao-contracts/issues/572